### PR TITLE
Added page for collecting user research references

### PR DESCRIPTION
**Pull request by GBKS**
[Preview](https://deploy-preview-381--sad-borg-390916.netlify.app/guide/designing-products/user-research/resources)

Based on #111. This adds a new page with the content of the user research reference document from #111, also linked [here](https://docs.google.com/document/d/1B3y0EnA7oaG9R_wrbddAOCJg7-PRMuyn41w1IVfnfBE/edit?usp=sharing).

I created this as a sub-page to the "User research" page in the "Designing Bitcoin products" section.

I've been collecting this research since January and asked today in Slack whether there's interest in turning it into a dedicated page. There was interest, so here we are.

The scope of this PR is just to port the content from the Google Doc to this new page. Additions, like reading through each research paper and summarizing findings should be done separately.

Setting this as a draft as I'll also try to come up with a header image. But feel free to provide feedback on the page content.

### CODE REVIEW

File: guide/designing-products/user-research-resources.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
+---
+layout: guide
+title: User research resources
```
Comment (ConorOkus) [inline guide/designing-products/user-research-resources.md:3]: Change to just "Resources" to match URL structure. Not a biggie though.

Comment (GBKS) [reply guide/designing-products/user-research-resources.md:3]: Wouldn't that be weird for people who randomly end up on this page, not knowing the context? Especially on mobile, you wouldn't see that the page is nested under a "User research" page.

File: guide/designing-products/user-research-resources.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+## Surveys
+
+###### [Project Horizon](https://youtu.be/oZkBU5H2WjY) (presentation recording)
+Presented in May 2021 by Jamaal Montasser.
+
+###### [Bitcoin Core Usage Survey](https://achow101.com/2021/01/bitcoin-core-survey)
+Launched in January 2021 by Andrew Chow. In progress.
+
+###### [A study on how Bitcoin is perceived by no-coiners (non-holders of Bitcoin)](https://github.com/MickMorucci/Bitcoin_perception)
+Launched in January 2021 by Mick Morucci. In progress.
```
[the remark this answers is not in the document]
Comment (GBKS) [reply guide/designing-products/user-research-resources.md:54]: Might be best to just exclude in-progress work. The Github repo has been stale for 6 months and who knows when it will get done.

File: guide/designing-products/user-research-resources.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+###### [The U in Crypto Stands for Usable: An Empirical Study of User Experience with Mobile Cryptocurrency Wallets](https://voskart.de/pdf/u_in_crypto.pdf)
+Published in January 2021 by Artemij Voskobojnikov, Oliver Wiese, Masoud Mehrabi Koushki, Volker Roth, and Konstantin Beznosov.
+
+###### [User Perception of Bitcoin Usability and Security across Novice Users](https://www.researchgate.net/publication/331286661_User_Perception_of_Bitcoin_Usability_and_Security_across_Novice_Users)
+Published in February 2019 by Abdulla Alshamsi & Keele University.
+
+###### [A First Look at the Usability of Bitcoin Key Management](https://arxiv.org/pdf/1802.04351.pdf)
+Published in February 2018 by Shayan Eskandari, David Barrera, Elizabeth Stobert, and Jeremy Clark.
+
+###### [An Evaluation of User Attitudes Towards Anonymity in Bitcoin](https://www.diva-portal.org/smash/get/diva2:1141672/FULLTEXT01.pdf)
```
[the remark this answers is not in the document]
Comment (GBKS) [reply guide/designing-products/user-research-resources.md:39]: Weird, works for me. Can you try again? I wonder if it's a country restriction.

**Comment by GBKS** [comment]
@ConorOkus good idea. We could start by asking Artemij, as he is in the design community. 

What do you think about adding a link for readers to submit more references, and another one to discuss in the user research Slack channel?

File: guide/designing-products/user-research-resources.md (hunk 1 above)
Comment (Bosch-0) [reply guide/designing-products/user-research-resources.md:3]: I like the idea of having this page just called 'Resources' and internally is split into:

1. Research
2. Tools (things designers may find useful) 
3. Other things like the Bitcoin Icon set / wallet UI Kit. 

Styling could be simple just with the Heading, such as 'Research' with an unordered list underneath linking externally to the resource.

Comment (GBKS) [reply guide/designing-products/user-research-resources.md:3]: I'd like to keep this PR focused on just putting together a decent page about user research. 

For future PRs, I'm personally more drawn to keeping this page focused on user research and adding good summaries of each project. It's pretty intimidating to see a 20-page academic paper so I think a lot of research is not being used. I think it's worth seeing if we cannot pull out the main findings and present them as TLDRs so they can be referenced by designers and maybe also serve as discussion material (on Slack and Twitter).

@Bosch-0 what are some examples of Tools? For your third point, the icon set is already linked to from the [Projects](https://bitcoin.design/projects/) page. There's also the [old concept of the design resource page](https://github.com/BitcoinDesign/Guide/issues/35). Those are good things to build out further, we just need to have clear places for each type of resource.

File: guide/designing-products/user-research-resources.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (ConorOkus) [reply guide/designing-products/user-research-resources.md:39]: Ok working for me now, the site was just probably down for a bit.

File: guide/designing-products/user-research-resources.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (ConorOkus) [reply guide/designing-products/user-research-resources.md:54]: Ok cool 👍

File: guide/designing-products/user-research-resources.md (hunk 1 above)
Comment (ConorOkus) [reply guide/designing-products/user-research-resources.md:3]: Yeah fair point, I'm in favour of the current format and then looking at ways to display the key findings in each piece of research.

**Comment by ConorOkus** [comment]
Yeah sounds good to me, maybe a short Google form could be okay. 
 - What is the research about?
 - Link to research paper
Linking to the slack channel would be good.

File: guide/designing-products/user-research-resources.md [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+   mobileRetina = "/assets/images/guide/designing-products/user-research-resources/user-research-resources-mobile@2x.png"
+   alt-text = "Printed Bitcoin survey results on a desk"
+   width = 1600
+   height = 600
+   layout = "full-width"
+%}
+
+# User research resources
+
+Below is a collection of published user research findings around Bitcoin, from academic papers, to surveys and blog posts.
```
Comment (pavlenex) [inline guide/designing-products/user-research-resources.md:35]:
```suggestion
Below is a collection of published user research findings around Bitcoin, from academic papers to surveys and blog posts.
```

**Review by pavlenex — COMMENTED** [review]
LGTM.

**Comment by Bosch-0** [comment]
LGTM, though It would look tidier / more professional if we cleaned up references using a citation style guide like APA - https://apastyle.apa.org/

File: guide/designing-products/user-research-resources.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (GBKS) [reply guide/designing-products/user-research-resources.md:73]: Put this one back in, as Mick now has posted an update.

**Comment by GBKS** [comment]
> LGTM, though It would look tidier / more professional if we cleaned up references using a citation style guide like APA - https://apastyle.apa.org/. Or at least listed these references in alphabetical order.

Why alphabetical? I tried to put the most useful information at the top, seems more useful.

Which one of those citation styles do you recommend? I looked at the site and it looks like there are lots of different formats to choose from (journal, magazine, newspaper, book...). TBH, I have a hard time with some of those formats they show ([example](https://apastyle.apa.org/style-grammar-guidelines/references/examples/published-dissertation-references)). It becomes really hard to parse, I find, as it's a bunch of facts stuffed into a single paragraph.

**Review by ConorOkus — APPROVED** [review]

**Comment by GBKS** [comment]
Added a follow-up issue for adding summaries here, see #401.

**Comment by Bosch-0** [comment]
> Why alphabetical? I tried to put the most useful information at the top, seems more useful.

Most useful based on what by who? Alphabetical order is a general standard for representing content such as this, makes it easier to sort / read. Not sure what information you are referring to here also, all I see is a title and authors not any information on the resource?

> Which one of those citation styles do you recommend?

APA is the style guide and the different formats are based on what kind of resource you are referencing (slightly different presentation for peer reviewed academic papers say compared to a published book). Examples for each kind of work can be found here: https://apastyle.apa.org/style-grammar-guidelines/references/examples. Using APA would also solve https://github.com/BitcoinDesign/Guide/issues/385.

**Comment by GBKS** [comment]
> Most useful based on what by who? Alphabetical order is a general standard for representing content such as this, makes it easier to sort / read. Not sure what information you are referring to here also, all I see is a title and authors not any information on the resource?

Sorry, totally forgot that I did order them by date. Newest ones at the top, so it's easier to see what's current and not accidentally read something from 2015 that may not be relevant anymore (considering this space evolves quickly).
